mappings/net/minecraft/world/chunk/Chunk.mapping

```diff
@@ -69,17 +72,19 @@ CLASS none/atp net/minecraft/world/chunk/Chunk
 	METHOD b getHeightAt (II)I
 		ARG 0 x
 		ARG 1 z
+	METHOD b setLastSavedTime (J)V
+		ARG 0 worldTime
 	METHOD b (Lnone/cn;)I
 		ARG 0 pos
 	METHOD b removeEntity (Lnone/sg;)V
 		ARG 0 entity
 	METHOD b update (Z)V
-	METHOD c addEntities ()V
+	METHOD c addEntitiesToWorld ()V
 	METHOD c setInhabitedTime (J)V
 		ARG 0 inhabitedTime
 	METHOD c setEntitiesLoaded (Z)V
 		ARG 0 value
-	METHOD d dropEntities ()V
+	METHOD d dropEntitiesFromWorld ()V
```

Comment:
Add instantly adds entities to the world, drop adds them to a list to be removed next tick.